src/platforms/javascript/common/troubleshooting/session-replay.mdx

```diff
@@ -0,0 +1,18 @@
+# Troubleshooting Replay specific issues
+
+This guides aims to extend the <PlatformLink to="/troubleshooting/">main troubleshooting</PlatformLink> guide covering Replay-specific scenarios.
+
+## My `canvas` elements are not getting captured.
+
+Currently there's not support for `canvas`. This feature is being tracked on this [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/6519).
+Please feel free to 👍 to help us prioritize it.
+
+##  My custom CSS/Images/Fonts/Media are not appearing when I view the replay
+
+The replay 'video' is actually a video-like reproduction of the HTML on your website. This means that all the external resources your site uses (CSS/Images/Fonts) will be rendered by the corresponding &lt;style&gt;, &lt;img&gt; and &lt;video&gt; tags on your site. 
+Add `sentry.io` to your CORS policy so the iframe hosted on sentry.io can fetch and display these resources. 
+
+## Replay on my browser extension doesn't work?
+
+This is not a supported scenario. The replay SDK is build to work on a website, and not on externally loaded script via browser extension or other mechanism.
+In fact, Sentry's Session Replay product can help developers find out when a third-party Chrome Extension causes issues with their websites which are otherwise hard to debug or reproduce.
```

Comment:
```suggestion
## Replay on My Bowser Extension Doesn't Work.

This is not a supported scenario. The replay SDK is built to work on a website and not as an externally-loaded script via browser extension or other mechanism. In fact, Sentry's Session Replay product can help developers find out when a third-party Chrome extension causes otherwise hard to debug or reproduce issues with their website.
```